src/classes/SObjectFieldWrapper.cls

```diff
@@ -0,0 +1,29 @@
+public class SObjectFieldWrapper {
```

Comment:
@jongpie I ended up renaming it (just now) to QueryField, made the constructors private, and made two static methods named "wrap" to initialize.  That's about as terse as I can imagine without something further lopping letters off the object name.